ios/Classes/NativeBridge/Handlers/BrowserHandler.swift

```diff
@@ -50,15 +50,15 @@ class BrowserViewController: NSObject, BrowserHandlerProtocol {
             Logger.log("webAuthSession completionHandler", sender: self)
             guard error == nil, let successURL = callbackURL else {
                 self.cancelButtonPressed()
-                return;
+                return
             }
 
             self.handleSuccessUrl(url: successURL)
         })
 
         if #available(iOS 13.0, *) {
```

Comment:
I guess this check is not needed since iOS 13 is currently minimum that SDK supports.

Reply:
lets not touch the browser logic in this PR; added it as a point within the browser register rework issue https://onewelcome.atlassian.net/browse/FP-35